### ISSUE TITLE
Lintly requests changes even if no issues were detected

### ISSUE THREAD

**Issue by phrdang** [grantmcconnaughey/Lintly#32]
Hello, thank you for making this GitHub action, it's been very useful!

I have the following workflow file:

```yml
name: Python (Lintly)

on: pull_request

jobs:
  lint-python:
    name: Lint Python with flake8
    runs-on: ubuntu-latest

    steps:
    - name: Check out Git repository
      uses: actions/checkout@v2

    - name: Set up Python
      uses: actions/setup-python@v1
      with:
        python-version: '3.x'
      
      # Install flake8 and Lintly
    - name: Install Python dependencies
      run: pip install flake8 lintly

      # Run Lintly with flake8
    - name: Lint with flake8
      run: flake8 | lintly --commit-sha=${{ github.event.pull_request.head.sha }}
      env:
        LINTLY_API_KEY: ${{ secrets.GITHUB_TOKEN }}
```

Oddly, the action requests changes even though there are no linting issues:

<img width="956" alt="Screen Shot 2020-08-04 at 1 32 27 PM" src="https://user-images.githubusercontent.com/35876322/89341961-f1c56b00-d656-11ea-810f-4cb7fee08da5.png">

Any help with this issue or a fix of this bug would be appreciated!

**Comment by grantmcconnaughey** [comment]
Hey there! Sorry for the late reply; this somehow fell off my radar. Is this PR in a public repo where I could take a look at the output?

**Comment by phrdang** [comment]
Yep, you can find it here: code4tomorrow/python#39. Brief background on the repository - it's being used as a collection of introductory Python code examples, problems, and solutions. Here's a link to the [workflow file](https://github.com/code4tomorrow/python/blob/master/.github/workflows/python-lint.yml) as well if you need it; it hasn't changed since I submitted this PR.

**Comment by abhishekkundalia** [comment]
I am facing the same issue. Please let me know if we found something.
